lib/Doctrine/ORM/Internal/Hydration/AbstractHydrator.php

```diff
@@ -351,13 +352,11 @@ protected function gatherScalarRowData(&$data)
 
             // WARNING: BC break! We know this is the desired behavior to type convert values, but this
             // erroneous behavior exists since 2.0 and we're forced to keep compatibility.
-            if ( ! isset($cacheKeyInfo['isScalar'])) {
-                $dqlAlias  = $cacheKeyInfo['dqlAlias'];
-                $type      = $cacheKeyInfo['type'];
-                $fieldName = $dqlAlias . '_' . $fieldName;
-                $value     = $type
-                    ? $type->convertToPHPValue($value, $this->_platform)
-                    : $value;
+            if (! isset($cacheKeyInfo['isScalar'])) {
```

Comment:
The change here now only make this more readable and removes the need for intermediate variables (which means fewer opcodes - not really relevant for modern PHP, though, thanks to DCE).

Reply:
Any idea why is this line part of diff? It looks same as before

Reply:
Whitespace before of `!` got removed to match our CS.